native/0001-Add-envoy_url-to-URLRequestContext.patch

```diff
@@ -167,40 +167,39 @@ index a5688c7f2bf9d..4468310d7b5a0 100644
 +      storage_path(storage_path),
 +      accept_language(accept_language),
 +      user_agent(user_agent),
-+      envoy_url(envoy_url),
```

Comment:
not sure how well i understand this but the envoy url being removed here jumped out at me.  do you have any context for this change?

Reply:
The `URLRequestContextConfig::URLRequestContextConfig` constructor is being overloaded with a second version that accepts an additional `envoy_url` param... so basically the definitions is copy/pasted with `envoy_url` added. patch barfed on it because of the Dict/DictStorage change, so I basically re-did the copy/paste. I think I put it above instead of below like the old patch... anyway, you can see that `envoy_url` is added to the other version of the function on line 191. It would be pretty trivial to move the code around so the diff is less messy if you care

Reply:
no need for changes if it's just cosmetic.